From: Phil Reid <preid@electromag.com.au>
Date: Tue, 28 Nov 2017 11:09:10 +0800
Subject: i2c: core: fix compile issue related to incorrect gpio header

Git-commit: 10c9ef045a7e19e9fd4c829c7321f9d2048808c0
Patch-mainline: v4.16-rc1
References: fate#326530,fate#326531,fate#326535,fate#326538,fate#326539

The correct header to include for the gpiod interface is
<linux/gpio/consumer.h>.

Fixes: 3991c5c80beaf7eb9 ("i2c: Switch to using gpiod interface for gpio bus recovery")
Signed-off-by: Phil Reid <preid@electromag.com.au>
Reviewed-by: Jarkko Nikula <jarkko.nikula@linux.intel.com>
Signed-off-by: Wolfram Sang <wsa@the-dreams.de>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/i2c/i2c-core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/i2c/i2c-core.c
+++ b/drivers/i2c/i2c-core.c
@@ -37,7 +37,7 @@
 #include <linux/delay.h>
 #include <linux/err.h>
 #include <linux/errno.h>
-#include <linux/gpio.h>
+#include <linux/gpio/consumer.h>
 #include <linux/hardirq.h>
 #include <linux/i2c.h>
 #include <linux/idr.h>
